users/test_views.py

```diff
@@ -1,8 +1,8 @@
-class TestViews:
-    def test_profile_view(self, client):
-        response = client.get('/users/profile/')
-        assert response.status_code == 302
+import pytest
 
-    def test_edit_profile_view(self, client):
-        response = client.get('/users/edit-profile/')
+
+class TestViews:
+    @pytest.mark.parametrize('view', ['', 'edit-profile/'])
+    def test_views(self, client, view):
+        response = client.get(f'/profile/{view}')
         assert response.status_code == 302
```

Comment:
I think you already test it in `test_redirections`

Reply:
> I think you already test it in `test_redirections`

Actually those tests were written in a merged PR, I just changed the urls.

Should.I delete this test and keep `test_redirections`?

Reply:
I think that if it's the same test, it should exist only once

Reply:
> I think that if it's the same test, it should exist only once

Done :)